packages/ra-ui-materialui/src/input/SelectArrayInput.stories.tsx

```diff
@@ -82,6 +83,150 @@ export const Basic = () => (
+                </Stack>
+            </SimpleForm>
+        </Create>
+    </AdminContext>
+);
+
+export const ReadOnly = (onSuccess = console.log) => (
+    <AdminContext i18nProvider={i18nProvider}>
+        <Create
+            resource="users"
+            record={{ roles: ['u001', 'u003'] }}
+            sx={{ width: 800 }}
+            mutationOptions={{ onSuccess }}
+        >
+            <SimpleForm>
+                <Stack direction="row">
+                    <Box>
+                        <SelectArrayInput
+                            source="roles"
+                            choices={[
+                                { id: 'admin', name: 'Admin' },
+                                { id: 'u001', name: 'Editor' },
+                                { id: 'u002', name: 'Moderator' },
+                                { id: 'u003', name: 'Reviewer' },
+                            ]}
+                            sx={{ width: 300 }}
+                            readOnly
+                        />
+                        <SelectArrayInput
+                            source="roles"
+                            variant="outlined"
+                            choices={[
+                                { id: 'admin', name: 'Admin' },
+                                { id: 'u001', name: 'Editor' },
+                                { id: 'u002', name: 'Moderator' },
+                                { id: 'u003', name: 'Reviewer' },
+                            ]}
+                            sx={{ width: 300 }}
+                            readOnly
+                        />
+                        <SelectArrayInput
+                            source="roles"
+                            variant="standard"
+                            choices={[
+                                { id: 'admin', name: 'Admin' },
+                                { id: 'u001', name: 'Editor' },
+                                { id: 'u002', name: 'Moderator' },
+                                { id: 'u003', name: 'Reviewer' },
+                            ]}
+                            sx={{ width: 300 }}
+                            readOnly
+                        />
+                    </Box>
+                    <Box>
+                        <SelectArrayInput
+                            source="title"
+                            sx={{ width: 300 }}
+                            readOnly
+                        />
+                        <SelectArrayInput
+                            source="title"
+                            variant="outlined"
+                            sx={{ width: 300 }}
+                            readOnly
+                        />
+                        <SelectArrayInput
+                            source="title"
+                            variant="standard"
+                            sx={{ width: 300 }}
+                            readOnly
+                        />
+                    </Box>
```

Comment:
**question**: there is no title, so why adding this?
```suggestion

```

Reply:
It's voluntary. It's a request from JB: to be able to see input behavior with and without data.